modules/plottinggl/src/plotters/scatterplotgl.cpp

```diff
@@ -464,6 +578,7 @@ void ScatterPlotGL::objectPicked(PickingEvent* p) {
     if (properties_.hovering_.get()) {
         if (p->getHoverState() == PickingHoverState::Enter) {
             hoverIndex_ = id;
+            hoverIndexGL_.upload(static_cast<const void*>(&id), sizeof(uint32_t));
```

Comment:
Might be bad doing opengl stfuff here.. might not have the right context. That is only garanted in process and initializeResourses, and the constructor.